### ISSUE TITLE
Getting 'could not parse requirement: UnknownRequirement("sparserevlog")' error

### ISSUE THREAD

**Issue by jalba69** [kilork/hg-git-fast-import#11]
Hello,

I was trying to use this tool that seems promising ( especially interested by the path_prefix option in toml config)  but i get an error no matter what i try.

> thread 'main' panicked at 'could not parse requirement: UnknownRequirement("sparserevlog")', src\libcore\result.rs:1165:5
stack backtrace:
   0:        0x13fc6be74 - <unknown>
   1:        0x13fb6b9eb - <unknown>
   2:        0x13fc6af24 - <unknown>
   3:        0x13fc6a93d - <unknown>
   4:        0x13fc69a07 - <unknown>
   5:        0x13fc70bc9 - <unknown>
   6:        0x13fc84f59 - <unknown>
   7:        0x13fb65a19 - <unknown>
   8:        0x13fb6c02f - <unknown>
   9:        0x13fa6b11f - <unknown>
  10:        0x13fab921e - <unknown>
  11:        0x13fa811cd - <unknown>
  12:        0x13fca115a - <unknown>
  13:        0x13fa7c2e4 - <unknown>
  14:        0x13fca1a55 - <unknown>
  15:         0x76fe556d - BaseThreadInitThunk
  16:         0x7714372d - RtlUserThreadStart

Command lines used : 
`hg-git-fast-import single RelativePathTo/local/x265hgRepo RelativePathTo/local/DestGitRepo`
`hg-git-fast-import single "RelativePathTo/local/x265hgRepo" "RelativePathTo/local/DestGitRepo"`
`hg-git-fast-import single "FullPathto/local/x265hgRepo" "Fullpathto/local/DestGitRepo"`

Same with command : 
`hg-git-fast-import single --source-pull="https://bitbucket.org/multicoreware/x265" "H:\x265" "H:\DestGitRepo"`
This one correctly pulls from URL then same error message: 

> pulling from https://bitbucket.org/multicoreware/x265
> searching for changes
> aucun changement trouvÚ
> thread 'main' panicked at 'could not parse requirement: UnknownRequirement("sparserevlog")', src\libcore\result.rs:1165:5

"_aucun changement trouvÚ_" , it's in French, means no changes found, while having a UFT8 encoding issue for last character "Ú" that should be "é"

Tried also variations with --clean --fix-wrong-branch-names but no luck
Also tried --log , a log file is created but it's empty.

Tried with no git repo already created, tried with one created and having git init done in it, tried with only a empty folder created for destination etc...

Bit out of ideas for now :/ 

Tried both the hg-git-fast-import-v1.3.0-x86_64-pc-windows-msvc binary and also one self compiled with cargo, always getting the same results.

Not sure if i'm overlooking something or if there is a bug.

Thanks a lot.

**Comment by kilork** [comment]
Hi, I know this issue, technically this means parser does not support sparserevlog (which is true).

As workaround: I know you can just edit .hg/requires and remove 'sparserevlog' from it. Give import a try, in all cases I tried - it works without issues.

As short term solution - I would add a cmd line options to ignore some requirements.

To properly solve it I need to support this requirement in parser.

**Comment by kilork** [comment]
Created a cross-reference issue https://github.com/kilork/hg-parser/issues/1

I would add a cli option in the context of this issue.

**Comment by jalba69** [comment]
Hello,

The trick to remove 'sparserevlog' worked, and yeah a CLI switch to ignore this will be nice, thank you.

Btw during my authors edition of the Single.toml file ( this will be painful when there will be more than a couple of authors :p ) i was prompted with a notification about UFT8 for names like "Léa".

I bypassed this by setting the encoding of the .toml file to uft8 and it worked.

Thanks a lot.
Closing the issue.
